src/Libraries/PythonNodeModelsWpf/PythonNode.cs

```diff
@@ -29,9 +31,32 @@ public void CustomizeView(PythonNode nodeModel, NodeView nodeView)
             dynamoViewModel = nodeView.ViewModel.DynamoViewModel;
             workspaceModel = nodeView.ViewModel.WorkspaceViewModel.Model;
 
-            var editWindowItem = new MenuItem { Header = PythonNodeModels.Properties.Resources.EditHeader, IsCheckable = false };
-            nodeView.MainContextMenu.Items.Add(editWindowItem);
-            editWindowItem.Click += delegate { EditScriptContent(); };
+            // If it is a Debug build, display a python engine switcher
+            if (dynamoViewModel.IsDebugBuild)
+            {
+                var editWindowItem = new MenuItem { Header = PythonNodeModels.Properties.Resources.EditHeader, IsCheckable = false };
+                nodeView.MainContextMenu.Items.Add(editWindowItem);
+                editWindowItem.Click += delegate { EditScriptContent(); };
+                var pythonEngineVersionMenu = new MenuItem { Header = PythonNodeModels.Properties.Resources.PythonNodeContextMenuEngineSwitcher, IsCheckable = false };
+                nodeView.MainContextMenu.Items.Add(pythonEngineVersionMenu);
+                pythonEngine2Item.Click += delegate { UpdateToPython2Engine(); }; var pythonEngine3Item = new MenuItem { Header = PythonNodeModels.Properties.Resources.PythonNodeContextMenuEngineVersionThree, IsCheckable = true };
```

Comment:
Can we have a newline at the end of the `;`? 
How is `pythonEngine3Item` a `var`? Isn't it hiding the private field?

Reply:
Yup, should be removed, I guess missed this